app/services/shopify_import/data_parsers/products/base_data.rb

```diff
@@ -24,7 +24,7 @@ def tags
 
         def option_types
           @option_types ||= @shopify_product.options.map do |option|
-            { option.name.strip.downcase => option_values(option.values) }
+            { option.name.strip => option_values(option.values) }
```

Comment:
This could result in problems in case Shopify's options are case sensitive. Perhaps keeping `option.name.strip.downcase` like I suggested in the issue, would work better?
https://github.com/spark-solutions/spree_shopify_importer/issues/31

Reply:
@vfonic Yes but in spree we have case-insensitive name validation for name. In creators i am downcasing names. Same as You suggested in the issue.

Reply:
This is parser, right?

Reply:
@vfonic Yes, it is just parsing data from shopify to spree.

Reply:
You're right. All option names coming from Shopify will be downcased. In case there are other option names created in Spree already, they won't match. Maybe you could add some tests to make sure this works as it should.

Reply:
@vfonic You had right with validation in case that OT/OV was created bu user not by import. Fixed